microapps/GeometryEditorApp/app/src/main/java/com/arcgismaps/toolkit/geometryeditorapp/screens/MainScreen.kt

```diff
@@ -0,0 +1,241 @@
+                ),
+                title = { Text("Geometry Editor App") },
+                actions = {
+                    Switch(
+                        checked = isDrawingEnabled,
+                        onCheckedChange = {
+                            isDrawingEnabled = if (isDrawingEnabled) {
+                                GraphicsOverlay().apply {
+                                    // add the sketched graphic to this graphics overlay
+                                    addSketchedGraphic(
+                                        geometryEditor = geometryEditor,
+                                        currentGraphicsOverlay = this
+                                    )
+                                    // update list of graphics overlays with this graphics overlay
+                                    graphicsOverlays.add(this)
+                                }
+                                // on checked change, stop the geometry editor
+                                stopGeometryEditor(geometryEditor)
+                                // set isDrawingEnabled to false
+                                false
+                            } else {
+                                // on checked change, start the geometry editor
+                                startGeometryEditor(geometryEditor)
+                                // set isDrawingEnabled to true
+                                true
+                            }
+                        })
+
+                    var actionsExpanded by remember { mutableStateOf(false) }
+                    IconButton(onClick = { actionsExpanded = !actionsExpanded }) {
+                        Icon(Icons.Default.MoreVert, "More")
+                    }
+
+                    GeometryEditorDropDownMenu(
+                        expanded = actionsExpanded,
+                        geometryEditor = geometryEditor,
+                        graphicsOverlays = graphicsOverlays,
+                        onDismissRequest = {
+                            actionsExpanded = false
+                        },
+                        onDrawingDisabled = {
+                            isDrawingEnabled = false
+                        }
+                    )
+                }
+            )
+        },
+    ) { innerPadding ->
+        MapView(
+            modifier = Modifier
+                .padding(innerPadding)
+                .fillMaxSize(),
+            arcGISMap = arcGISMap,
+            geometryEditor = geometryEditor,
+            graphicsOverlays = graphicsOverlays
+        )
+    }
+}
+
+/**
+ * A dropdown menu to provide options for the [geometryEditor]
+ */
+@Composable
+fun GeometryEditorDropDownMenu(
+    modifier: Modifier = Modifier,
+    expanded: Boolean = false,
+    geometryEditor: GeometryEditor,
+    onDismissRequest: () -> Unit = {},
+    graphicsOverlays: GraphicsOverlayCollection,
+    onDrawingDisabled: () -> Unit = {}
```

Comment:
Seems like a more descriptive parameter name could be:
```suggestion
    onResetAllGraphics: () -> Unit = {}
```